.travis.yml

```diff
@@ -20,3 +26,11 @@ matrix:
     - env: NAME="lint"
       script:
         - yarn lint
+    - env: NAME="test"
+      install:
+        - export DISPLAY=':99.0'
+        - sh -e /etc/init.d/xvfb start
+        - sleep 3
```

Comment:
There's the `xvfb-maybe` module which can make this simpler.

You can exclude all the above and then:
```
  "test": "xvfb-maybe mocha ./dist/test"
```
And then it will sort out xvfb if its ever required

Reply:
thx :)